python/src/FunctionalChaosAlgorithm_doc.i.in

```diff
@@ -202,7 +282,58 @@ Test it:
 >>> print(model(X))
 [0.239713]
 >>> print(metamodel(X))
-[0.239514]"
+[0.239514]
+
+There are several methods to define the algorithm: default 
+settings are used when the information is not provided by the user.
+The simplest is to set only the input and output samples.
+In this case, the distribution and its parameters are estimated from
+the `inputSample` using the :class:`~openturns.MetaModelAlgorithm.BuildDistribution` class.
```

Comment:
```suggestion
the *inputSample* using the :class:`~openturns.MetaModelAlgorithm.BuildDistribution` class.
```
inputSample is a parameter so we use italics (at least this seems to be the convention in other API doc pages).